### PR TITLE
add cron example

### PR DESCRIPTION
**Pull request by KuthumiPepple**
@delawen Cron example runs successfully.

### CODE REVIEW

**Comment by Delawen** [comment]
Can you rename the folder to `13-cron`?

Also, can you add a bit more context on what the route does? Like here https://github.com/apache/camel-k-examples/tree/main/12-aws-secret-manager-vault At least add the command line to run it.

**Comment by KuthumiPepple** [comment]
> Can you rename the folder to `13-cron`?
> 
> Also, can you add a bit more context on what the route does? Like here https://github.com/apache/camel-k-examples/tree/main/12-aws-secret-manager-vault At least add the command line to run it.

I've renamed and added a didact tutorial. Let me know if more changes are needed.

**Review by oscerd — APPROVED** [review]

**Review by squakez — COMMENTED** [review]
I think Didact is going to be discontinued. Its repo was archived not long time ago: https://github.com/redhat-developer/vscode-didact

**Comment by squakez** [comment]
I don't like having number in front of the directory either. I prefer having an index as we used to do in the Camel K examples: https://github.com/apache/camel-k/tree/main/examples

**Comment by KuthumiPepple** [comment]
> I think Didact is going to be discontinued. Its repo was archived not long time ago: https://github.com/redhat-developer/vscode-didact

Hmm. I didn't know that. Makes sense why new examples didn't use it

**Comment by KuthumiPepple** [comment]
> I don't like having number in front of the directory either. I prefer having an index as we used to do in the Camel K examples: https://github.com/apache/camel-k/tree/main/examples

I like the index too. I'll speak to my mentor about it.

**Comment by Delawen** [comment]
Whatever @squakez says, we do :)

**Comment by squakez** [comment]
> Whatever @squakez says, we do :)
> 
> (except re-writing the whole Camel K in Javascript, there must always be red lines!)

I am flattered :) However, my opinion counts as anybody's one else. I may be wrong so we better have open discussion and submit to vote when no agreement is reached.

**Comment by KuthumiPepple** [comment]
@squakez what's the difference between custom and generic examples? In the root folder's readme, I see examples grouped under these categories.

**Comment by oscerd** [comment]
I think it makes sense to remove them. We should have just generic and kamelets.

**Comment by KuthumiPepple** [comment]
Cleaned up commit history. Improved cron example in generic-examples directory.

File: generic-examples/cron/README.md [hunk 1]
```diff
@@ -1,3 +1,105 @@
...
+
+**Apache Camel K CLI ("kamel")**
+You need the Apache Camel K CLI ("kamel") in order to access all Camel K features.
+
+### Optional Requirements
+
+The following requirements are optional. They don't prevent the execution of the demo, but may make it easier to follow.
+
+**VS Code Extension Pack for Apache Camel**
+The VS Code Extension Pack for Apache Camel provides a collection of useful tools for Apache Camel K developers,
```
Comment (apupier) [inline generic-examples/cron/README.md:24]:
```suggestion
The [VS Code Extension Pack for Apache Camel](https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack) provides a collection of useful tools for Apache Camel K developers,
```

Comment (KuthumiPepple) [reply generic-examples/cron/README.md]: Done. Thanks

**Comment by KuthumiPepple** [comment]
@tadayosi I think this one can be merged too

**Review by Delawen — APPROVED** [review]

**Pull request merged**